airflow/models/taskinstance.py

```diff
@@ -742,6 +742,34 @@ def error(self, session=NEW_SESSION):
         session.merge(self)
         session.commit()
 
+    @classmethod
+    @provide_session
+    def find(
```

Comment:
This function is only used for this one test, right? If so lets not add it (I'm not sure the pattern we have for DagRun.find etc is _good_)

Reply:
Ah its used one place --  but still, I don't think we need it.